frontend/src/lib/components/ActivationSidebar/ActivationLogic.ts

```diff
@@ -0,0 +1,238 @@
+        skipTask: (id: string) => ({ id }),
+        setShowSessionRecordingConfig: (value: boolean) => ({ value }),
+    }),
+    reducers({
+        skippedTasks: [
+            [] as string[],
+            { persist: true, prefix: CACHE_PREFIX },
+            {
+                skipTask: (state, { id }) => [...state, id],
+            },
+        ],
+        showSessionRecordingConfig: [
+            false,
+            {
+                setShowSessionRecordingConfig: (_, { value }) => value,
+            },
+        ],
+        isReady: [
+            false,
+            {
+                loadCustomEventsSuccess: () => true,
+                loadCustomEventsFailure: () => true,
+            },
+        ],
+    }),
+    loaders(({ cache }) => ({
+        customEventsCount: [
+            0,
+            {
+                loadCustomEvents: async ({}, breakpoint) => {
+                    breakpoint(200)
+                    const url = api.eventDefinitions.determineListEndpoint({
+                        event_type: EventDefinitionType.EventCustom,
+                    })
+                    if (url in (cache.apiCache ?? {})) {
+                        return cache.apiCache[url]
+                    }
+                    cache.eventsStartTime = performance.now()
+                    const response = await api.get(url)
+                    breakpoint()
+                    cache.apiCache = {
+                        ...(cache.apiCache ?? {}),
+                        [url]: response.count,
+                    }
+                    return cache.apiCache[url]
+                },
+            },
+        ],
+    })),
+    selectors({
+        tasks: [
+            (s) => [s.currentTeam, s.members, s.invites, s.customEventsCount, s.installedPlugins, s.skippedTasks],
+            (currentTeam, members, invites, customEventsCount, installedPlugins, skippedTasks) => {
+                const tasks: Task[] = []
+                for (const task of Object.values(ACTIVATION_TASKS)) {
+                    switch (task) {
+                        case ACTIVATION_TASKS.INGEST_FIRST_EVENT:
+                            tasks.push({
+                                id: ACTIVATION_TASKS.INGEST_FIRST_EVENT,
+                                name: 'Ingest your first event',
+                                description: 'Ingest your first event to get started with PostHog',
+                                completed: currentTeam?.ingested_event ?? false,
+                                canSkip: false,
+                                skipped: false,
+                            })
+                            break
+                        case ACTIVATION_TASKS.INVITE_TEAM_MEMBER:
+                            tasks.push({
+                                id: ACTIVATION_TASKS.INVITE_TEAM_MEMBER,
+                                name: 'Invite a team member',
+                                description: 'Every person in your company can benefit from PostHog',
```

Comment:
Users may not be in a company. They could be in some other sort of organization. We could say team? But then that may be too limiting.

```suggestion
                                description: 'Everyone in your organization can benefit from PostHog',
```